code/game/jobs/job/_job.dm

```diff
@@ -37,6 +37,7 @@
 
 	var/announced = TRUE                  //If their arrival is announced on radio
 	var/latejoin_at_spawnpoints           //If this job should use roundstart spawnpoints for latejoin (offstation jobs etc)
+	var/forced_spawnpoint                 //If this job should spawn on specific spawnpoint only
```

Comment:
1. Comment is confusing, it's implying that it's a flag, but it's a list of landmarks?
2. Nothing is actually setting it anywhere.

Reply:
1. It can be a list, or just text string which must reference to a real spawn point. I'll edit this message later then.
2. Because I don't want make it for tradeship right now. Conceptually we can spawn captain in his room, instead regular cryo. I can add this as feature later in this PR, just don't want mess with map right now.

Reply:
it's passed to pick() so it's confusing. Comment implies it's a flag either way pls change.

My point is there's no landmark /type/ that sets it, how are you going to set a /job datum var/ on map?